amundsen_application/static/js/components/TableDetail/OpenRequestMetadata/index.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import './styles.scss';
+
+import { GlobalState } from 'ducks/rootReducer';
+import { connect } from 'react-redux';
+import { ToggleRequestAction } from 'ducks/notification/types';
+import { toggleRequest } from 'ducks/notification/reducer';
+import { bindActionCreators } from 'redux';
+
+interface StateFromProps {
+  requestIsOpen: boolean;
+}
+
+export interface DispatchFromProps {
+  toggleRequest: () => ToggleRequestAction;
+}
+
+export type OpenRequestMetadataProps = StateFromProps & DispatchFromProps;
```

Comment:
nit nit: I think "RequestDescription" is a more succinct and appropriate name.

Reply:
gonna keep this as openrequestmetadata to keep naming consistent with RequestMetadataForm's name

Reply:
You might end up re-considering when we add the other components. More than one type of component will open this modal, so what do they get called? Why would this one be called `OpenRequestMetadata` as opposed to the others? In this sense, you might find it better to name the components even more specifically -- in this case we could  name it according to the feature it serves.

However this is still a nit, let's see what we can do with names down the road.